Don't install files already installed by pinocchio
Add pinocchio dependency

--- CMakeLists.txt~	2025-08-12 16:55:44.130129334 +0200
+++ CMakeLists.txt	2025-08-12 16:55:56.513226916 +0200
@@ -11,7 +11,6 @@
 )
 set(PROJECT_URL "http://github.com/stack-of-tasks/pinocchio")
 set(PROJECT_CUSTOM_HEADER_EXTENSION "hpp")
-set(PROJECT_USE_CMAKE_EXPORT TRUE)
 set(PROJECT_USE_KEYWORD_LINK_LIBRARIES TRUE)
 set(PROJECT_COMPATIBILITY_VERSION AnyNewerVersion)
 set(PINOCCHIO_PROJECT_SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR})
@@ -210,6 +209,7 @@
 # ----------------------------------------------------
 set(CMAKE_MODULE_PATH "${JRL_CMAKE_MODULES}/find-external/CppAD/" ${CMAKE_MODULE_PATH})
 add_project_dependency(Eigen3 REQUIRED PKG_CONFIG_REQUIRES "eigen3 >= 3.0.5")
+add_project_dependency(${PROJECT_NAME} REQUIRED)
 
 if(BUILD_WITH_URDF_SUPPORT)
   add_project_dependency(urdfdom_headers REQUIRED)
@@ -518,7 +518,4 @@
 pkg_config_append_cflags("${CFLAGS_DEPENDENCIES}")
 pkg_config_append_cflags("${CFLAGS_OPTIONS}")
 
-# Install catkin package.xml
-install(FILES package.xml DESTINATION share/${PROJECT_NAME})
-
 setup_project_finalize()
